exe/bwa_mqtt_bridge

```diff
@@ -395,7 +395,7 @@ class MQTTBridge
     end
 
     @homie["spa"]["current-temperature"].hass_sensor(device_class: :temperature)
-    @homie["spa"]["target-temperature"].hass_number(icon: "mdi:thermometer")
+    @homie["spa"]["target-temperature"].hass_number(icon: "mdi:thermometer", step: 0.5)
```

Comment:
this is only true if the temperature_scale is celsius; can you please make this conditional